src/Microsoft.ML.AutoML/ColumnInference/ColumnGroupingInference.cs

```diff
@@ -8,6 +8,8 @@
 using Microsoft.ML.Data;
 using static Microsoft.ML.Data.TextLoader;
 
+using Range = Microsoft.ML.Data.TextLoader.Range;
```

Comment:
Is this necessary

Reply:
Yes it is because we also have a class called Range inside ML.NET and it gets confused otherwise.